bigframes/ml/decomposition.py

```diff
@@ -27,7 +28,15 @@
 import bigframes.pandas as bpd
 import bigframes.session
 
-_BQML_PARAMS_MAPPING = {"svd_solver": "pcaSolver"}
+_BQML_PARAMS_MAPPING = {
+    "svd_solver": "pcaSolver",
+    "feedback_type": "feedbackType",
+    "num_factors": "numFactors",
+    "user_col": "userColumn",
+    "item_col": "itemColumn",
+    # TODO: Add rating_col
```

Comment:
Are we blocked on something for this TODO? If so, please add context. If not, let's add support for this.

Edit: Look at https://cloud.google.com/bigquery/docs/reference/rest/v2/models#trainingoptions for the corresponding parameters.

Edit2: I found userColumn and itemColumn but nothing corresponding to rating column. I think this is similar to that lower bounds issue (bug 391399223).

Edit3: I found in the code that `rating_col` gets written to `inputLabelColumns`, which will be a list of columns (just one if set in this case). I think we might need to do something with `@property` to solve this difference. Something like the following:

```suggestion
    "_input_label_columns": "inputLabelColumns",
```

and then below a `@property` for `rating_col` that looks at `_input_label_columns` and returns the first one if set, otherwise `None`.